pandas/tests/extension/base/ops.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy as np
+import pandas as pd
+from .base import BaseExtensionTests
+
+
+class BaseOpsTests(BaseExtensionTests):
+    """Various Series and DataFrame ops methos."""
+
+    def check_op(self, s, op, other, exc=NotImplementedError):
+
+        with pytest.raises(exc):
+            getattr(s, op)(other)
+
+    def test_arith_scalar(self, data, all_arithmetic_operators):
+        # scalar
+        op = all_arithmetic_operators
+        s = pd.Series(data)
+        self.check_op(s, op, 1, exc=TypeError)
```

Comment:
maybe use `s[0]` as the scalar? (so to have the same type) That might be more general likely to work.